docs/dev/developer-guides/aa.md

```diff
@@ -36,13 +30,10 @@ The account abstraction protocol on zkSync is very similar to [EIP4337](https://
 
 ### Keeping nonces unique
 
-::: warning Changes are expected
-
-The current model has some important drawbacks: it does not allow custom wallets to send multiple transactions at the same time, while keeping a deterministic ordering. For
-EOAs the nonces are expected to be sequentially growing, while for the custom accounts the order of transactions can not be determined for sure.
-
-In the future, we plan to switch to a model where the accounts could choose whether they would want to have sequential nonce ordering (the same as EOA) or they want to have arbitrary ordering.
-
+::: warning
+- The current model does not allow custom wallets to send multiple transactions at the same time, while keeping a deterministic ordering. 
+- For EOAs, nonces are expected to grow sequentially; while for custom accounts the order of transactions cannot be guaranteed.
+- In the future, we plan to switch to a model where accounts can choose between sequential or arbitrary nonce-ordering.
```

Comment:
```suggestion
- In the future, we plan to switch to a model where accounts can choose between sequential or arbitrary nonce-ordering. 
- The function `updateNonceOrdering` is already available for this in the `Account` constructor and should be called along with `updateAccountVersion`.
```

Reply:
@StanislavBreadless please check my added bullet for this in relation to this ticket: https://linear.app/matterlabs/issue/DOC-197/[allopay]-set-accountnonceordering

Reply:
@kailash-manasarovar it is correct, but since the functionality is not supported on the server side, let's skip it from the docs

Reply:
OK ignoring suggestion.